### DISCUSSION TITLE
exercise 6 Q1

### DISCUSSION THREAD

**Discussion by emilyyli** [LiNk-NY/IntroR#43] [Q&A]
Hi @ralythompson24, just wanted to make sure the data set is coming from mtcars from exercise 6, question #1 .

Also for "What type of motor has a higher mean miles per gallon?", do you just want us to print our code results and compare the mpg or write a code in R comparing the mpg for each motor? 

<img width="877" alt="image" src="https://github.com/CUNYSPHCode/IntroR/assets/171090659/9a3a82dc-4ab2-4371-8772-63a9a6f3c38b">
<img width="852" alt="image" src="https://github.com/CUNYSPHCode/IntroR/assets/171090659/c8d742f6-b774-42b9-8142-9b980dc2a474">

**Comment by ralythompson24** [comment]
Hi @emilyyli - thanks for your question. Yep, you should be using the `mtcars` data here, as you have done. You have the first part complete, but you should also write another code chunk where you calculate the means for each group (not doing a statistical test though). I think the easiest way to accomplish this is using `dplyr` functions, we did something similar in Session 4 Exercise!

**Reply by emilyyli** [reply]
Oh I see thank you! so it would become something like this below?

<img width="644" alt="image" src="https://github.com/CUNYSPHCode/IntroR/assets/171090659/bf636f50-1be0-4e63-8f98-1fdde33cbad9">

**Reply by LiNk-NY** [reply]
That looks good @emilyyli 
Just make sure that you copy and paste the text rather than attaching images. The text is easier to reproduce :)

**Reply by CourtA918** [reply]
> Hi @emilyyli - thanks for your question. Yep, you should be using the `mtcars` data here, as you have done. You have the first part complete, but you should also write another code chunk where you calculate the means for each group (not doing a statistical test though). I think the easiest way to accomplish this is using `dplyr` functions, we did something similar in Session 4 Exercise!


@ralythompson24 I was actually about to ask a similar question regarding this because the question did not give clarity on if we would utilize mtcar. if I solely did this t.test( mpg, vs, data= mtcars) would this be the correct way?

**Reply by ralythompson24** [reply]
Hi @CourtA918, almost but you actually need to use a formula to define the t-test, like this `t.test(mpg ~ vs, data = mtcars`.

The formula goes like this:
`y ~ x`, where `y` is the numeric variable (what you're calculating the mean for) and `x` defines the groups you are comparing the means across. 

https://www.rdocumentation.org/packages/stats/versions/3.6.2/topics/t.test
